### ISSUE TITLE
Accessibility (a11y) guidelines

### ISSUE THREAD

**Issue by sloria** [CenterForOpenScience/osf-style#74]
Add tips and guidelines for making our style accessible to users with disabilities.

**Comment by sloria** [comment]
...and with this, a plan for QA to test for a11y.

**Comment by caneruguz** [comment]
@sloria I agree wholeheartedly of course, but becoming accesible is a matter of strategic direction too, for instance to what extent we support it and in what level etc. Someone needs to decide that and we need to start going through the website to comply with it. It needs to be a concious decision, otherwise we would just be patching things here and there but can't claim an accesible experience for our users.

**Comment by sloria** [comment]
As much as buttons, colors, interactions, and content are usability concerns, so is accessibility. We don't need to make an exception to our process. For the time being, we can implement these changes just as we have been for other guidelines: (1) Deciding what the guidelines should be, (2) Enforcing the guidelines for all new development, and (3) Transitioning old code when we have the opportunity.

**Comment by caneruguz** [comment]
100% agree with this. 
Pinging @JeffSpies , @lyndsysimon , @lbanner, @GaryKriebel for thoughts on how to implement this plan and whether it's a priority.

**Comment by GaryKriebel** [comment]
This looks to be a candidate for the next epic.

**Comment by sloria** [comment]
@caneruguz Let's stick with our usual process--You decide on guidelines and send a PR; you and I sign off on them; and we follow those guidelines moving forward. As far as transitioning old code: big changes = proposal; small fixes = PR.

While others are welcome to comment, remember that you are owner of this guide, and you can protect your and others' time by going through the right communication channels, according to our process.

**Comment by JeffSpies** [comment]
+1 @sloria on process. 

On the substance of this one, I'd like to reach out to folks at ARL. There are some specific guidelines that Universities are required to follow by some top-down mandate that we'll want to make sure is consistent. I don't know if that's a11y or not. ARL said before that they'd be happy to help us when we were interested.

I'll let you know what I hear.

**Comment by GaryKriebel** [comment]
Excellent plan on ARL input as well.

**Comment by JeffSpies** [comment]
"[Start with the] Web Content Accessibility Guidelines (http://www.w3.org/TR/WCAG20/), which is the standard/specification to which we encourage libraries to hold their vendors to, and it is the standard that the federal guidelines (section 508 of the Rehabilitation Act; http://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/proposed-rule) are being refreshed to align with."

**Comment by caneruguz** [comment]
Thanks @JeffSpies , good input to start work on this! 
I'll add an accesibility page after my current work is done.

**Comment by caneruguz** [comment]
This is a great talk I attended at the Converge conference. I would recommend to go through the slides:
https://docs.google.com/presentation/d/15UqDivt4y-AUm-wfelHpC7L8MBCT6V9_vZQYXDXwWPQ/pub?start=false&amp;loop=false&amp;delayms=3000&slide=id.p
